docs/project-lifecycle.md

```diff
@@ -0,0 +1,123 @@
+[//]: # (SPDX-License-Identifier: CC-BY-4.0)
+
+# Project Lifecycle
+
+The term "project" within the Hyperledger Project will refer to a
```

Comment:
`the Hyperledger Project` -> `Hyperledger`

Reply:
Still seeing `the Hyperledger Project`

Reply:
fixed.